app/src/main/java/eu/kanade/tachiyomi/util/chapter/ChapterTrackSync.kt

```diff
@@ -40,3 +53,72 @@ fun syncChaptersWithTrackServiceTwoWay(db: DatabaseHelper, chapters: List<Chapte
         }
     }
 }
+
+private var job: Job? = null
+private var oldMangaId: Long? = null
+
+/**
+ * Starts the service that updates the last chapter read in sync services. This operation
+ * will run in a background thread and errors are ignored.
+ */
+fun Controller.updateTrackChapterMarkedAsRead(
+    db: DatabaseHelper,
+    preferences: PreferencesHelper,
+    oldLastChapter: Chapter?,
+    newLastChapter: Chapter?,
+    mangaId: Long?,
+    delay: Long = 3000
```

Comment:
What is the purpose of the callback here ?

Reply:
Honestly I forgot the full reason by now lol. I think it was to update trackers in manga details page, since that page shows tracking it needs to be updated after it finishes calling the sevices